docs/conversion_examples_gallery/imaging/femtonics.rst

```diff
@@ -0,0 +1,40 @@
+Femtonics imaging data conversion
+---------------------------------
+
+Install NeuroConv with the additional dependencies necessary for reading Femtonics data.
+
+.. code-block:: bash
+
+    pip install "neuroconv[femtonics]"
+
+Convert Femtonics imaging data to NWB using :py:class:`~neuroconv.datainterfaces.ophys.femtonics.femtonicsimaginginterface.FemtonicsImagingInterface``
+
+.. code-block:: python
+    >>> import sys, platform
+    >>> from zoneinfo import ZoneInfo
+    >>> from pathlib import Path
+    >>> from neuroconv.datainterfaces import FemtonicsImagingInterface
+    >>>
+    >>> file_path = OPHYS_DATA_PATH / "imaging_datasets" / "Femtonics" / "moser_lab_mec" / "p29.mesc"
+    >>>
+    >>> # Check available sessions, units, and channels
```

Comment:
I would prefer to not have those methods in the conversion gallery. The first entry in the conversion gallery should showcase the simplicity of conversion instead of all the functionality. If you still believe this is is very important we can add a second section to the conversion gallery but I don't think this is necessary.